security-admin/src/main/java/org/apache/ranger/rest/PublicAPIsv2.java

```diff
@@ -745,18 +745,20 @@ public RangerRole getRole(@PathParam("id") Long id, @Context HttpServletRequest
     	This API is used to add users and groups with/without GRANT privileges to this Role. It follows add-or-update semantics
  	 */
 	@PUT
-	@Path("/api/roles/{id}/addUsersAndGroups")
-	public RangerRole addUsersAndGroups(@PathParam("id") Long roleId, List<String> users, List<String> groups, Boolean isAdmin, @Context HttpServletRequest request) {
-		return roleREST.addUsersAndGroups(roleId, users, groups, isAdmin);
```

Comment:
@ted12138  - thank you for the patch. The changes look good.

One suggestion: to be consistent with use of` {id}` and `{name}` in APIs, I would suggest to retain `{id}` in this API and introduce another API for using `{name}` - like: `/api/roles/name/{name}/addUserAndGroups`.

Reply:
OK, I got your point, I will solve it later.